src/main/java/com/student/user/app/controllers/item/UpdateStudentController.java

```diff
@@ -151,5 +161,8 @@ protected void onUpdateStudOClicked(ActionEvent event){
         }
 
     }
+    private boolean isValidEmail(String email) {
+        return email != null && email.contains("@") && email.endsWith(".com");
```

Comment:
use regular expressions (regex) to validate email addresses since this code will reject email addresses with domains other than ".com" such as ".org" and ".edu" and etc.